pkg/heartbeat/app.go

```diff
@@ -90,5 +87,40 @@ func GetHeartbeatInfo(sdkStore store.Store) *types.HeartbeatInfo {
 		}
 	}
 
+	if sdkStore.GetAdditionalMetricsEndpoint() != "" {
+		additionalMetrics, err := getAdditionalMetrics(sdkStore.GetAdditionalMetricsEndpoint(), sdkStore.GetLicense().Spec.LicenseID)
+		if err != nil {
+			logger.Errorf("failed to get additional metrics: %v", err.Error())
```

Comment:
hmm, only logging an error and continuing to report without the additional metrics can be problematic. let's discuss.